### PR TITLE
Several pretty minor edits after I read through this doc.

### PR DESCRIPTION
**Pull request by adeinega**

### CODE REVIEW

File: draft-ietf-gnap-resource-servers.md [hunk 1]
```diff
@@ -100,8 +100,8 @@ Terminology specific to GNAP is defined in the terminology section of the core s
 When the AS issues an access token for use at an RS, the RS
 needs to have some means of understanding what the access token is for
 in order to determine how to respond to the request. The core GNAP
-protocol makes no assumptions or demands on the format or contents
-of the access token, and in fact the token format and contents are opaque
+protocol neither makes assumptions nor demands on the format or contents
```
Comment (jricher) [inline draft-ietf-gnap-resource-servers.md:103]:
```suggestion
protocol makes neither assumptions nor demands on the format or contents
```

File: draft-ietf-gnap-resource-servers.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ to be used at one or more RSs. The AS can offer an introspection service
 to allow an RS to validate that a given access token:
 
  - has been issued by the AS
- - has not expired
+ - has not been expired
```
Comment (jricher) [inline draft-ietf-gnap-resource-servers.md:226]: Reverting as "expiration" is usually a passive action based on a timeout.

```suggestion
 - has not expired
```

File: draft-ietf-gnap-resource-servers.md [hunk 3]
```diff
@@ -76,14 +76,14 @@ connections are particularly useful for connecting an AS and RS from
...
-This specification defines several means for a RS and AS to
+This specification defines several means for an RS and AS to
 communicate these aspects with each other, including structured
 access tokens and RS-facing APIs. This specification also discusses
 methods for an RS to derive a downstream token for calling another
 chained RS.
 
 The means of the authorization server issuing
-the access token to the client instance and the means of the client instance
+the access token to the client instance, and the means of the client instance
```
Comment (jricher) [inline draft-ietf-gnap-resource-servers.md:86]: Only two items here, no comma needed.

```suggestion
the access token to the client instance and the means of the client instance
```

**Review by jricher — CHANGES_REQUESTED** [review]
Thanks for the read through! It's appreciated. A couple comments on this, and marking it editorial for when we pull it in.

**Comment by adeinega** [comment]
A couple of other minor things

> key (object/string): : REQUIRED if the token is bound. The key bound to the access token, to allow the RS to validate the signature of the request from the client instance. If the access token is a bearer token, this MUST NOT be included.

it sounds to me like an optional step for the RS to validate the signature if the key object is presented in the introspection response. That should be MUST, right?

The third step in section "Token Introspection" states that

> The AS validates the access token value and the Resource Server's request and returns the introspection response for the token.

I think that the AS should validate the RS's request and only after that the AT's value.

**Comment by jricher** [comment]
The RS always has to validate the signature, but sometimes it can't fully validate it until after it looks up the token's data using introspection. We should look at what OAuth MTLS and DPoP say in terms of RS validation steps.

The intent of the language for the AS validation was not to imply an order in either direction, so we can look to tweak this to make that more clear.

**Review by jricher — APPROVED** [review]

**Pull request merged**